developer_setup/running_test_suites.md

```diff
@@ -1,33 +1,30 @@
 ## Running the test suites
 
-ManageIQ specs are split into several components:
-
-* **vmdb**
-* **automation**
-* **migrations**
-* **providers**
-* **replication**
-* **javascript**
-* **brakeman**
-
-Each suite has a pair of Rake tasks to setup and run them, in the form of `test:<type>` and
-`test:<type>:setup` (if required):
-
-```bash
-rake test:vmdb:setup             # Setup environment for vmdb specs
-rake test:vmdb                   # Run all specs except migrations, replication, and automation
-rake test:automation:setup       # Setup environment for automation specs
-rake test:automation             # Run all automation specs
-rake test:migrations:setup       # Setup environment for migration specs
-rake test:migrations             # Run all migration specs
-rake test:providers:amazon:setup # Setup environment for Amazon provider specs
-rake test:providers:amazon       # Run all Amazon provider specs
-rake test:replication:setup      # Setup environment for replication specs
-rake test:replication            # Run all replication specs
-rake test:javascript:setup       # Setup environment for javascript specs
-rake test:javascript             # Run all javascript specs
-rake test:brakeman               # Run Brakeman
-```
+Each ManageIQ component (providers, UI, vmdb, automation engine, ...) contains
+its own set of specs. Most of them can be run by simply executing
+
+    $ bundle exec rake
+
+Components that require non-standard invocation of test suite are documented
+in the remaining part of the document.
+
+
+### Running the vmdb suite
+
+Before the vmdb suite can be run, test database needs to be set up. In vast
+majority of cases, test database is created as a part of a `bin/update`, so
+there is little need to call `bundle exec rake test:vmdb:setup`. But it is
+there if you feel you need it.
+
```

Comment:
Is there an actual situation where the test database wouldn't be created if you run bin/setup? Also, I would generally replace "vmdb" with "ManageIQ".

"The ManageIQ test database needs to be setup before the test suite can be run. This is automatically created if you run `bin/setup`, but can also be created using `bundle exec rake test:vmdb:setup`."

Reply:
No, `bin/setup` and `bin/update` will always create test database if they succeed. I will remove the offending part from the doc. As for the vmdb, I used that name in order to be consistent with the rest of the document.